PowerValidatedSolutions.psm1

```diff
@@ -20714,6 +20721,384 @@ Function Get-PasswordPolicyConfig {
+                                $encounterError = "True"
+                            }
+                            if (($PasswdPolicyRetryValue -lt 0) -or ($PasswdPolicyRetryValue -gt 9999)) {
+                                Write-LogMessage -Type ERROR -Message "The recommended range for retry should be between 0 and 9999"  -Colour Red
+                                $encounterError = "True"
+                            }
+                            if ((($PasswdPolicyMinValue1 -lt 7) -or ($PasswdPolicyMinValue1 -gt 999)) -and ($PasswdPolicyMinValue1 -ine "disabled")) {
+                                Write-LogMessage -Type ERROR -Message "The recommended policy setup should be retry=3 min=disabled,disabled,disabled,disbled,15"  -Colour Red
+								Write-LogMessage -Type ERROR -Message "The custom policy file shows $policyString"  -Colour Red
+                                $encounterError = "True"
+                            }
+                            elseif ((($PasswdPolicyMinValue2 -lt 7) -or ($PasswdPolicyMinValue2 -gt 999)) -and ($PasswdPolicyMinValue2 -ine "disabled")) {
+                                Write-LogMessage -Type ERROR -Message "The recommended policy setup should be retry=3 min=disabled,disabled,disabled,disbled,15"  -Colour Red
+								Write-LogMessage -Type ERROR -Message "The custom policy file shows $policyString"  -Colour Red
+                                $encounterError = "True"
+                            }
+                            elseif ((($PasswdPolicyMinValue3 -lt 7) -or ($PasswdPolicyMinValue3 -gt 999)) -and ($PasswdPolicyMinValue3 -ine "disabled")) {
+                                Write-LogMessage -Type ERROR -Message "The recommended policy setup should be retry=3 min=disabled,disabled,disabled,disbled,15"  -Colour Red
+								Write-LogMessage -Type ERROR -Message "The custom policy file shows $policyString"  -Colour Red
+                                $encounterError = "True"
+                            }
+                            elseif ((($PasswdPolicyMinValue4 -lt 7) -or ($PasswdPolicyMinValue4 -gt 999)) -and ($PasswdPolicyMinValue4 -ine "disabled")) {
+                                Write-LogMessage -Type ERROR -Message "The recommended policy setup should be retry=3 min=disabled,disabled,disabled,disbled,15"  -Colour Red
+								Write-LogMessage -Type ERROR -Message "The custom policy file shows $policyString"  -Colour Red
+                                $encounterError = "True"
+                            }
+                            elseif ((($PasswdPolicyMinValue5 -lt 7) -or ($PasswdPolicyMinValue5 -gt 999)) -and ($PasswdPolicyMinValue5 -ine "disabled")) {
+                                Write-LogMessage -Type ERROR -Message "The recommended policy setup should be retry=3 min=disabled,disabled,disabled,disbled,15"  -Colour Red
+								Write-LogMessage -Type ERROR -Message "The custom policy file shows $policyString"  -Colour Red
```

Comment:
Change to:

Write-LogMessage -Type ERROR -Message "Password Policy Configuration File Defined as ($policyString)"  -Colour Red